crates/revm/src/evm_impl.rs

```diff
@@ -586,42 +583,84 @@ impl<'a, GSPEC: Spec, DB: Database + 'a, const INSPECT: bool> Host
         scheme: CreateScheme,
         value: U256,
         init_code: Bytes,
-        gas: u64,
+        gas_limit: u64,
     ) -> (Return, Option<H160>, Gas, Bytes) {
         if INSPECT {
-            let (ret, gas, bytes, out) =
-                self.inspector
-                    .create(&mut self.data, caller, &scheme, value, &init_code, gas);
+            let (ret, address, gas, out) = self.inspector.create(
+                &mut self.data,
+                caller,
+                &scheme,
+                value,
+                &init_code,
+                gas_limit,
+            );
             if ret != Return::Continue {
-                return (ret, gas, bytes, out);
+                return (ret, address, gas, out);
             }
         }
-        self.create_inner::<SPEC>(caller, scheme, value, init_code, gas)
+        let (ret, address, gas, out) =
```

Comment:
A note here, it seems that `out` is always `Bytes::empty`?

Reply:
It most cases it is. Only here it returns something: https://github.com/bluealloy/revm/blob/c0bab703000e125a81b2ef35210af55a56d5664f/crates/revm/src/evm_impl.rs#L398